commit c13c9945bf84817ee1dd2d65e58660fb38e23a7f
Author: wm4 <wm4@nowhere>
Date:   Sun Mar 13 15:52:17 2016 +0100

    player: add missing audio reconfig events
    
    This also takes care of sending the required property change
    notifications.
    
    Fixes #2929 and maybe fixes #2920.

diff --git a/player/audio.c b/player/audio.c
index f17587a..3a2c60b 100644
--- a/player/audio.c
+++ b/player/audio.c
@@ -134,6 +134,8 @@ static int recreate_audio_filters(struct MPContext *mpctx)

     mixer_reinit_audio(mpctx->mixer, mpctx->ao, afs);

+    mp_notify(mpctx, MPV_EVENT_AUDIO_RECONFIG, NULL);
+
     return 0;

 fail:
@@ -368,6 +370,8 @@ static void reinit_audio_filters_and_output(struct MPContext *mpctx)

     update_playback_speed(mpctx);

+    mp_notify(mpctx, MPV_EVENT_AUDIO_RECONFIG, NULL);
+
     return;

 init_error:
